donkeycar/templates/cfg_complete.py

```diff
@@ -94,7 +94,7 @@
 #The DEFAULT_MODEL_TYPE will choose which model will be created at training time. This chooses
 #between different neural network designs. You can override this setting by passing the command
 #line parameter --type to the python manage.py train and drive commands.
-DEFAULT_MODEL_TYPE = 'linear'   # (linear|categorical|tflite_liner|tensorrt_linear)
+DEFAULT_MODEL_TYPE = 'linear'   #(linear|categorical|rnn|imu|behavior|3d|localizer|latent)
 BATCH_SIZE = 128                #how many records to use when doing one pass of gradient decent. Use a smaller number if your gpu is running out of memory.
 TRAIN_TEST_SPLIT = 0.8          #what percent of records to use for training. the remaining used for validation.
```

Comment:
We don't have the any of the models `rnn|imu|behavior|3d|localizer|latent`

Set  it to `linear|categorical|tflite_liner|tensorrt_linear`

Reply:
Yep, merge error on my side.